pages/common/starship-init.md

```diff
@@ -0,0 +1,20 @@
+# starship init
+
+> Print the integration code for the infinitely customizable prompt for any shell.
```

Comment:
Readers of `starship init` will probably already know what `starship` itself is.
```suggestion
> Print shell integration code for starship.
```